src/main/java/com/nutomic/syncthingandroid/fragments/DeviceSettingsFragment.java

```diff
@@ -97,13 +103,43 @@ public void onCreate(Bundle savedInstanceState) {
                 mDevice.Name = "";
                 mDevice.DeviceID = "";
                 mDevice.Addresses = "dynamic";
-                mDevice.Compression = true;
+                mDevice.Compression = "always";
```

Comment:
I'm pretty sure reducing communication time saves more battery than reducing CPU load, so I suggest to keep this.